src/App.jsx

```diff
@@ -0,0 +1,32 @@
+import { Routes, Route, Navigate } from "react-router-dom";
+import { Home } from "./pages/Home";
+import { Store } from "./pages/Store";
+import { Wishlist } from "./pages/Wislist";
+import { Cart } from "./pages/Cart";
+import { Login } from "./pages/Login";
+import { Profile } from "./pages/Profile";
+import { ProductDetail } from "./pages/ProductDetail";
+import { NavBar } from "./components/NavBar";
+import { Snakbar } from "./components/Snakbar";
+import { ProtectedRoute } from "./components/ProtectedRoute";
+import { useSnakbarContext } from "./Context/SnakbarContext";
+// import { useAuthContext } from "./Context/AuthContext";
+
+export default function App() {
+  const { snakbarStatus } = useSnakbarContext();
+  return (
+    <main>
+      <NavBar></NavBar>
+      <Routes>
```

Comment:
You can create a separate file for your routes, so that as your app grows, maintaining that file becomes easy and also helps in refactoring.